log.txt

```diff
@@ -0,0 +1 @@
+--PERFORMANCE LOG START --
```

Comment:
The log file should be added to the .gitignore, as we don't need it or its contents in the repo